__tests__/bundle/primitive-types/snapshot.js

```diff
@@ -30,7 +30,7 @@ paths:
                   decimal: 12.34
                   numberWithLeadingZero: 127
                   boolean: true
-                  date: 2020-01-01
+                  date: '2020-01-01'
```

Comment:
why?

Reply:
because of that https://github.com/Redocly/openapi-cli/pull/523